README.md

```diff
@@ -53,6 +54,7 @@ Website for findeasily.net
 - [x] User login
 - [x] Password reset with Email
 - [x] User profile page
+    - [x] Update profile picture
 - [ ] User can create/edit/delete listing info
```

Comment:
how about user can view list page, and user will be unable to edit/delete/update the lists others created

Reply:
It is a permission related issue, which will be handled soon. ;-)